subsys/llext/llext_link.c

```diff
@@ -142,6 +143,206 @@ static const void *llext_find_extension_sym(const char *sym_name, struct llext *
 	return se.addr;
 }
 
+/**
+ * @brief Determine address of a symbol.
+ *
+ * @param link_addr (output) resolved address
+ * @param rel relocation entry
+ * @param sym symbol entry
+ * @param ext llext extension
+ * @param ldr llext loader
+ * @param name symbol name
+ * @param shdr section header
+ * @param section_idx section index
+ * @param entry_idx entry index
+ *
+ * @return 0 for OK, negative for error
+ */
+static int llext_lookup_symbol(uintptr_t *link_addr, const elf_rela_t *rel, const elf_sym_t *sym,
+			       struct llext *ext, struct llext_loader *ldr, const char *name,
+			       const elf_shdr_t *shdr, int section_idx, int entry_idx)
+{
+	*link_addr = 0;
```

Comment:
this isn't in the original code and is superfluous - as long as you add an error check in the caller

Reply:
Removed